libclc/utils/gen-libclc-test.py

```diff
@@ -13,11 +13,30 @@
 
 def ignore_function(fun):
     whitelist = [
+<<<<<<< HEAD
```

Comment:
This should go away after rebasing to #1828.

Reply:
I fixed this